cypress/integration/v2/features/global-settings/global-colors/index.js

```diff
@@ -308,6 +310,14 @@ export function globalColorNativeBlocks() {
 
 		nativeBlocks.forEach( blockName => {
 			cy.addBlock( blockName )
+			if ( blockName !== 'core/separator' && blockName !== 'core/cover' ) {
+				cy
+					.get( `.wp-block[data-type='${ blockName }']` )
+					.type( 'Block Title', { force: true } )
```

Comment:
```suggestion
				cy
					.get( `.wp-block[data-type='${ blockName }']` )
					.type( 'Block Title', { force: true } )
```
```suggestion
				cy
					.get( `.wp-block[data-type='${ blockName }']` )
					.type( 'Block Title', { force: true } )
```

why are we not using `wp.typeBlock` here?

Reply:
okay will change this